### ISSUE TITLE
Duplicates when shooting non dead soldiers

### ISSUE THREAD

**Issue by x3003** [Realm667/WolfenDoom#940]
When trying out the release version w/ the score, more weapons and behaviour add-ons (yes, some seems to work together) when you shoot some almost dead soldiers, they duplicate. Weird...

![Screenshot_BoA_20210710_235634](https://user-images.githubusercontent.com/77861029/125177948-61859380-e1e0-11eb-94bc-29c1b81a5431.png)

![Screenshot_BoA_20210711_000534](https://user-images.githubusercontent.com/77861029/125177950-664a4780-e1e0-11eb-9cb1-aed33e64a75a.png)

**Comment by Ozymandias81** [comment]
Addons must be loaded alone, we didn't tested mixes and that's why. I am sure it comes from sleeping soldiers.

**Comment by MuadDib90** [comment]
You can load multiple addons at once. But I only tried to mix meaningful ones. Nothing serious to report though back then.
However this exactly I did not try to do.

**Comment by Tormentor667** [comment]
That’s the point. I guess we need to deactivate loading multiple addons. This only leads to issues.

**Comment by MuadDib90** [comment]
I must agree. Otherwise we would need a hell of a time to test it and find bugs, which there are as we see it. And also to adjust the code, which won't be as easy as it may seem (to me at least).

**Comment by x3003** [comment]
> I am sure it comes from sleeping soldiers.

No, they aren't sleepers. You first shoot them and when they get up to shoot you, you shoot them. And then they duplicate sometimes. Could be just 1 add-on which causes this, not multiple.

**Comment by x3003** [comment]
With **only** the behaviour addon this is reproducable. Sometimes they duplicate b4 they get up.

![Screenshot_BoA_20210711_133156](https://user-images.githubusercontent.com/77861029/125193486-e0b0b100-e24c-11eb-91bb-f9bfd352e5fa.png)

![Screenshot_BoA_20210711_133339](https://user-images.githubusercontent.com/77861029/125193491-e6a69200-e24c-11eb-95dd-3a2e5591a815.png)

**Comment by Ozymandias81** [comment]
That happens because the random check for half_dead spawning happens 2 times in a row probably through replaces inheritances. Doesn't change the fact that we must stitch to only one addon per Campaign.

**Comment by x3003** [comment]
>Doesn't change the fact that we must stitch to only one addon per Campaign.

As said above, it also happens when you **only** have the behaviour add on set.

**Comment by Tormentor667** [comment]
I will summon @Username-N00b-is-not-available here as he was creating the Behaviour addon setting. Furthermore we definitely need to deactivate loading multiple addons at once. Otherwise things will get out of hand considering all the combinations we have to deal with.
